src/witan/cic/episodes.clj

```diff
@@ -22,7 +22,7 @@
 
 (def scrubbed-episode-file-header
   [[::row-id ::id ::reason-new-episode ::report-date ::ceased ::legal-status ::care-status ::placement ::report-year ::sex ::dob ::uasc ::period-id ::episode-number ::phase-number ::phase-id]
-   ["" "ID" "RNE" "report_date" "ceased" "legal_status" "care_status" "placement" "report_year" "sex" "DOB" "SEX" "UASC" "period_id" "episode_number" "phase_number" "phase_id"]])
+   [""        "ID" "RNE"               "report_date" "ceased" "legal_status" "care_status" "placement" "report_year" "sex" "DOB" "UASC" "period_id" "episode_number" "phase_number" "phase_id"]])
```

Comment:
???

Reply:
that's just me trying to line up the 2 vectors so that I can see that they are matching